Kitodo/src/main/java/org/kitodo/production/forms/createprocess/AdditionalDetailsTableRow.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.forms.createprocess;
+
+import java.io.Serializable;
+import java.lang.reflect.Method;
+import java.util.Collection;
+import java.util.EnumMap;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.kitodo.api.MdSec;
+import org.kitodo.api.Metadata;
+import org.kitodo.api.dataeditor.rulesetmanagement.Domain;
+import org.kitodo.api.dataformat.IncludedStructuralElement;
+import org.kitodo.exceptions.InvalidMetadataValueException;
+import org.kitodo.exceptions.NoSuchMetadataFieldException;
+
+public abstract class AdditionalDetailsTableRow implements Serializable {
```

Comment:
Why is this class called `...TableRow` ?
Please avoid reflecting the frontend usage in the classname